app/helpers.js

```diff
@@ -0,0 +1,11 @@
+exports.paginated = (data, params) => {
+  const { offset = 0, limit = 5 } = params;
+  return data.slice(offset, offset + limit);
+};
+
+exports.order = (data, sortKey, sortOrder) => {
+  const sorting = sortOrder === 'ASC' ? -1 : 1;
```

Comment:
Move this assignment to a constant file, i.e: `exports.ASC = 1`;

Also you can create an enum in graphql to define sorting order input.

Reply:
All suggested changes are ok, thank you very much